.github/workflows/scratch-org-sfdx-ci-develop.yml

```diff
@@ -0,0 +1,106 @@
+# Unique name for this workflow
+name: Salesforce DX CI (scratch org)
+
+# Definition when the workflow should run
+on:
+    push:
+        branches:
+            - develop
+        paths-ignore:
+            - 'sfdx-project.json'
+            - 'README.md'
+    pull_request:
```

Comment:
@adityanaag3 we need roll back the workflow merge you suggested earlier and split that into two separate workflows: one for master pushes and one for master PRs.

Unfortunately, this is the only way to have a clean badge status displayed on the readme. If we don't do that the readme badge will report errors when a PR fails. The branch filter on the badge URL doesn't prevent that.

Reply:
Done